dts/bindings/misc/nxp,s32-emios.yaml

```
@@ -2,7 +2,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 description: |
-  NXP S32 Enhanced Modular IO SubSystem (eMIOS) node for S32 SoCs.
+  NXP S32 Enhanced Modular IO SubSystem (eMIOS) for S32 SoCs.
```

```suggestion
  Enhanced Modular IO SubSystem (eMIOS) for NXP S32 SoCs.
```
if you want to also rephrase this would be okay, same for the other `nxp,s32-*` bindings touched on this pr.

---

Looks good.